src/main/java/org/gradle/profiler/InvocationSettings.java

```diff
@@ -330,6 +338,11 @@ public InvocationSettingsBuilder setStudioInstallDir(File studioInstallDir) {
             return this;
         }
 
+        public InvocationSettingsBuilder setStudioSandboxDir(File studioSandboxDir) {
```

Comment:
Should this be `@Nullable`?

Reply:
Add @Nullable. Note that all properties here are nullable, so that brings some inconsistency, but maybe we will use @Nullable for future additions.